extensions/css-language-features/README.md

```diff
@@ -4,18 +4,20 @@ This extension offers CSS/SCSS/Less support in VS Code.
 
 ## Development
 
-- `yarn`
-- `yarn` at `/server`
-- Run the `Launch Extension` debug target in the Debug View
-- Run `Reload Window` command in the launched instance to reload the extension
+- Clone the `vscode` repository
+- Open the `extensions/css-language-features/` directory in VS Code.
+- Run `yarn` in `css-language-features/`.
+- Run `yarn` in `css-language-features/server/`.
+- Run the `Launch Extension` debug target in the Debug View.
+- Run `Reload Window` command in the launched instance to reload the extension.
 
 ### Try vscode-css-languageservice
 
 [`vscode-css-languageservice`](https://github.com/Microsoft/vscode-css-languageservice) contains the language smarts for CSS/SCSS/Less. This extension only wraps the service into a Language Server for VS Code.
 
 If you want to fix CSS/SCSS/Less issues or make improvements, you can:
 
-- Clone `vscode-css-languageservice` and compile it
-- Run `npm link` in `vscode-css-languageservice`
-- In extension, run `npm link vscode-css-languageservice` in `/server`
-- Now when you run `Launch Extension`, the launched instance will use your version of `vscode-css-languageservice`.
\ No newline at end of file
+- Clone `vscode-css-languageservice` and compile it with `npm compile`.
```

Comment:
It may be a good idea to clarify what steps from the first section need to be taken before getting to this part?

Reply:
Sure, I'll add some more instruction.